sympy/series/tests/test_limits.py

```diff
@@ -738,6 +738,11 @@ def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
 
 
+def test_issue_18452():
+    assert limit(abs(log(x))**x, x, 0, "+") == 1
```

Comment:
This without `+` was required.

Reply:
> By default the `dir` will be `+` in this case, so the testcase having `dir` explicitly specified as `+` can be removed.

As I have mentioned here.

Reply:
aren't both the same?

Reply:
Since they are same, that's why `+` is not required. That will be there by default.

Reply:
I have corrected that.